pkg/operator/baremetal_pod.go

```diff
@@ -98,7 +98,7 @@ func createMariadbPasswordSecret(client coreclientv1.SecretsGetter, config *Oper
 	return err
 }
 
-func newMetal3Deployment(config *OperatorConfig) *appsv1.Deployment {
+func newMetal3Deployment(config *OperatorConfig, baremetalConfig BaremetalConfig) *appsv1.Deployment {
```

Comment:
I feel like I must be missing something. This reads the config from a new CRD so that it can pass it to this function, but then it appears to not be used.

Reply:
@bison yes, that is coming in a following PR.

Reply:
Using it is coming in a subsequent PR. Adding TODO to indicate that in code.